src/main/java/org/candlepin/subscriptions/jmx/TallyJmxBean.java

```diff
@@ -87,11 +89,37 @@ public void tallyAccountByHourly(
     tasks.tallyAccountByHourly(accountNumber, tallyRange);
   }
 
-  @ManagedOperation(description = "Trigger hourly tally for all configured accounts")
-  public void tallyAllAccountsByHourly() {
-    log.info(
-        "Hourly tally for all accounts triggered over JMX by {}", ResourceUtils.getPrincipal());
+  @ManagedOperation(
+      description =
+          "Trigger hourly tally for all configured accounts for the specified range. The 'start' and "
+              + "'end' parameters MUST be specified as a pair to complete a range. If they are left empty, "
+              + "a date range is used based on NOW with the configured offsets applied (identical to if "
+              + " the job was run).")
+  @ManagedOperationParameter(
+      name = "start",
+      description = "The start date for the tally. Must be specified along with the end parameter.")
```

Comment:
Do you mind putting an example string as part of these parameter descriptions?  For example, in `MeteringJmxBean.java` we have "i.e 2018-03-20T09:00:00Z" in the description.  It's made it super convenient for copy & paste purposes.